### PR TITLE
Add set_bool method to account

### PR DESCRIPTION
**Pull request by xshill**

### CODE REVIEW

**Comment by xshill** [comment]
@mlefebvre review pls. Is there anything else I need to change?

**Comment by mlefebvre** [comment]
I think you also need to modify account.pyi

Maybe also add a test next to test_account_set_string!

File: purple/account.pyi [hunk 1]
```diff
@@ -32,9 +32,11 @@ class Account:
     def get_protocol_id(self) -> Optional[bytes]: ...
     def get_protocol_name(self) -> bytes: ...
     def get_string(self, name: bytes, default_value: bytes) -> bytes: ...
+    def get_bool(self, name: bytes, default_value: bool) -> bool: ...
     def is_enabled(self, ui_name: Optional[bytes] = None) -> bool: ...
     def set_password(self, password: bytes) -> None: ...
     def set_string(self, name: bytes, value: bytes) -> None: ...
+    def set_string(self, name: bytes, value: bool) -> None: ...
```
Comment (mlefebvre) [inline purple/account.pyi]: :thinking:

Comment (xshill) [reply purple/account.pyi]: omg

**Comment by mlefebvre** [comment]
lgtm

**Pull request merged**